app/routes/$.tsx

```diff
@@ -53,18 +52,7 @@ export const loader = async ({ request, params }: Route.LoaderArgs) => {
   handleRedirects(new URL(request.url).pathname);
   handleStaticFileRequests(params["*"]);
```

Comment:
Probably can move this stuff to middleware at some point, but I'd rather do that in another PR